internal/osvdev/config.go

```diff
@@ -0,0 +1,27 @@
+package osvdev
+
+import "github.com/google/osv-scanner/internal/version"
+
+type ClientConfig struct {
+	MaxConcurrentRequests      int
+	MaxConcurrentBatchRequests int
+
+	MaxRetryAttempts           int
+	JitterMultiplier           float64
+	BackoffDurationExponential float64
+	BackoffDurationMultiplier  float64
+	UserAgent                  string
+}
+
+// Default make a default client config
+func DefaultConfig() ClientConfig {
+	return ClientConfig{
+		MaxRetryAttempts:           4,
+		JitterMultiplier:           2,
+		BackoffDurationExponential: 2,
+		BackoffDurationMultiplier:  1,
+		UserAgent:                  "osv-scanner-v2-" + version.OSVVersion,
```

Comment:
shall we just use `osv-scanner-v` as prefix here?

Reply:
ah yep, meant to change this as this was just for testing, I think in osv.dev we'll update it to osv-go-client or something, and have osv-scanner override it. I'll change this back to our current one for now.